operators/test/e2e/stack/checks_k8s.go

```diff
@@ -317,3 +322,38 @@ func CheckESPassword(stack Builder, k *helpers.K8sHelper) helpers.TestStep {
 		}),
 	}
 }
+
+// KibanaDoReq executes an HTTP request against a Kibana instance.
+func KibanaDoReq(k *helpers.K8sHelper, stack Builder, method string, uri string, body []byte) ([]byte, error) {
```

Comment:
Maybe `KibanaReq` or `DoKibanaReq`? (`KibanaDoReq` feels like Kibana is doing the request).

Reply:
`DoKibanaReq`